src/app/routes/_app/rulesets/$rulesetSlug/rulebooks/$rulebookSlug/edit.module.css

```
@@ -0,0 +1,163 @@
+.workspaceSticky {
+  --workspace-top: calc(var(--app-shell-header-offset, 0px) + 5.5rem);
+  --workspace-height: clamp(28rem, calc(100dvh - var(--app-shell-header-offset, 0px) - 7rem), 52rem);
+  --preview-caption-space: 2.5rem;
+  --fit-height-page-height: calc(var(--workspace-height) - var(--preview-caption-space));
+  --fit-height-page-width: calc(var(--fit-height-page-height) * 210 / 297);
+  container: rulebook-editor-workspace / inline-size;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)